pretyping/reductionops.ml

```diff
@@ -468,23 +468,6 @@ let safe_meta_value sigma ev =
   try Some (Evd.meta_value sigma ev)
   with Not_found -> None
 
-let strong_with_flags whdfun flags env sigma t =
-  let push_rel_check_zeta d env =
```

Comment:
That's quite sneaky! Doesn't it just shout for having a new delta_let binder to control let-body expansion?

Reply:
It's only used in `cbn` now, and I am not touching `cbn` with a 10-foot pole. That code will sit nicely in its own sealed module like nuclear waste in geological strata.

Reply:
I would like to have such a flag on user-land though...